src/components/CategoryPicker/index.js

```diff
@@ -34,21 +34,36 @@ function CategoryPicker({policyCategories, reportID, iouType, iou, policyRecentl
         ];
     }, [iou.category]);
 
+    const sections = useMemo(() => {
```

Comment:
Disabled category is not indicated - https://github.com/Expensify/App/issues/36485#issuecomment-1943884263